cookbooks/bcpc/recipes/bootstrap_network.rb

```diff
@@ -73,6 +73,12 @@
   line "supersede routers #{router};"
 end
 
+replace_or_add 'append search domain' do
```

Comment:
Ah yes, because it's not in a `Berksfile` but a `metadata.rb`:
```vagrant@bcpc-bootstrap:~/chef-bcpc$ find cookbooks/ -name 'metadata*' -exec grep line \{\} \+
cookbooks/bcpc/metadata.rb:depends 'line'```

And as you point out, `replace_or_add` is used just above this block. All looks good